packages/core/src/util/field.ts

```diff
@@ -138,3 +139,18 @@ export const defaultMapStateToEnumFieldProps = (state, ownProps): StatePropsOfEn
  */
 export const mapDispatchToFieldProps: (dispatch) => DispatchPropsOfControl =
   mapDispatchToControlProps;
+
+/**
+ * Default map dispatch to enum field props to customize handleChange action
+ *
+ * @param dispatch
+ * @param ownProps
+ */
+export const defaultMapDispatchToEnumFieldProps = (dispatch, ownProps) => {
```

Comment:
Isn't this actually more generic in the sense that this is `defaultMapDispatchToControlProps`?

Reply:
Yes, you are right. User can overwrite `handleChange` method for all fields. I will change it